### ISSUE TITLE
R wrapper

### ISSUE THREAD

**Issue by noamross** [rmcantin/bayesopt#3]
I realize this may be out-of-scope, but it's just out of my abilities to contribute myself and I hope a community member will take it up.  An R wrapper for BayesOpt would be great.  With Rcpp it should be (relatively) easy to wrap the C++ API, much like [nloptr](https://github.com/jyypma/nloptr) does for NLopt.

**Comment by rmcantin** [comment]
I totally agree, but my knowledge of R is minimal and I rather focus on adding new core features. If  anyone wants to write the wrapper, I'll be glad to integrate it.
